src/main/java/com/stosik/parking/ParkingController.java

```diff
@@ -0,0 +1,55 @@
+package com.stosik.parking;
+
+import com.stosik.parking.reservation.domain.ReservationFacade;
+import com.stosik.parking.reservation.dto.CreateReservationCommand;
+import com.stosik.parking.reservation.dto.ReservationDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+@RestController
+@RequestMapping("/parking")
+@RequiredArgsConstructor
+class ParkingController
+{
+    private final ReservationFacade reservationFacade;
+    
+    @PostMapping(value = "/driver/start")
+    public ReservationDto startParkmeter(@RequestBody CreateReservationCommand createReservationCommand)
+    {
+        return reservationFacade.startParkmeter(createReservationCommand);
+    }
+    
+    @PutMapping("/driver/stop")
+    public ReservationDto stopParkmeter(@RequestParam String licenseId)
+    {
+        return reservationFacade.stopParkmeter(licenseId);
+    }
+    
+    @GetMapping("/driver/cost")
+    public BigDecimal dispendReservationCost(@RequestParam Long reservationId)
+    {
+        return reservationFacade.dispendReservationTicket(reservationId);
+    }
+    
+    @GetMapping("/operator/control")
```

Comment:
czy to nie powinny być 2 oddzielne kontrollery?

Reply:
Faktycznie, wydaje mi się że lepiej byłoby oddzielić controllery w zależności od ról (osób w systemie) i pełnionych przez nie funkcji.